src/test/groovy/com/infusionsoft/gradle/version/GitVersionResolverTest.groovy

```diff
@@ -44,7 +52,7 @@ class GitVersionResolverTest extends RepositoryTestCase {
     @Test
     void testGetVersionExistingReleaseWithUncommittedChanges() {
         String tagVersion = '0.0.1'
-        String expectedVersion = tagVersion + defaultTagOptions.snapshotSuffix
+        String expectedVersion = '0.0.2' + defaultTagOptions.snapshotSuffix
```

Comment:
my thought (validated by @frossbeamish) is that once you have version x.y.z tagged, any future changes (committed or otherwise) should not be considered version x.y.z-SNAPSHOT, but should be version x.y.(z+1)-SNAPSHOT.